### ISSUE TITLE
Błąd Strefy planistyczne - UnboundLocalError: local variable 'jestOk' referenced before assignment 

### ISSUE THREAD

**Issue by Edixn98** [CyfryzacjaAPP/wtyczka_qgis_app_2#7]
Przygotowałam strefy planistyczne poza wtyczką APP. Atrybuty uzupełnione były zgodnie ze schematem, geometria została poprawiona. Podjęto próbę wklejenia ich na warstwę StrefyPlanistyczne we wtyczce i zapisania pliku do gml. Pojawił się poniższy błąd. Z czego on wynika?

![obraz](https://github.com/CyfryzacjaAPP/wtyczka_qgis_app_2/assets/173901688/1b280c79-b3e3-4d78-81ba-a71d6db4272d)

Wersja eksperymentalna z 16 kwietnia - 2.0.3

**Comment by Edixn98** [comment]
W poprzedniej eksperymentalnej wersji wtyczki w ten sam sposób przygotowane dane udało się  zapisać do pliku gml.

**Comment by KasiaZagrobelna** [comment]
Dzień dobry, czy wszystkie obiekty w załączanej warstwie mają unikalne oznaczenia i identyfikatory?

**Comment by Edixn98** [comment]
tak, nawet jak robię to na niewielkiej próbie to pojawia się błąd. 
przykładowy plik gml z uzupełnionymi atrybutami który chcę przykleić do warstwy do wtyczki: 
[StrefaPlanistyczna.zip](https://github.com/user-attachments/files/15988823/StrefaPlanistyczna.zip)
Plik już po wklejeniu geometrii, w formacie gpkg, którego nie mogę zapisać do gml
[StrefaPlanistyczna_20.zip](https://github.com/user-attachments/files/15988847/StrefaPlanistyczna_20.zip)

**Comment by Edixn98** [comment]
podane przeze mnie błędy nadal występują w wersji wtyczki 2.0.4

**Comment by CyfryzacjaAPP** [comment]
W danych zamieszczonych w zgłoszeniu wszystkie obiekty mają zakończony cykl życia - prosimy o zmianę i informację, jeśli problem nadal występuje. Wraz z informacją prosimy o przesłanie plików oraz używanej wersji Wtyczki APP2 oraz programu QGIS.
